### ISSUE TITLE
Main dialog sometimes not shown on Windows

### ISSUE THREAD

**Issue by bllngr** [nextcloud/desktop#2966]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Clicking the tray icon or selecting "Open main dialog" always shows the main dialog in the respective corner, without exception.

### Actual behaviour
On one of my Windows 10 machines, the icon in the Windows task bar appears, but the main dialog is not shown. No matter what I do, I cannot get the main overlay to be shown.

Removing the `geometry` setting from `nextcloud.cfg` doesn't help either.

### Steps to reproduce
Unfortunately, I have no idea how to reproduce it :(

### Client configuration
Client version: 3.1.3

Operating system: Windows 10

OS language: en-US

Installation path of client: <default>

### Logs

1. Client logfile: 
https://gist.github.com/bllngr/a2418bc370daeea82d313884f75b3ae6

2. Web server error log:
Not applicable

3. Server logfile: nextcloud log (data/nextcloud.log):
Not applicable

**Comment by bllngr** [comment]
Well, there hasn't been any release, and the issue continues to persist. @FlexW, is there anything I need to do to not let the issue go stale? Do you need more information? Should I try to find out if older versions work correctly? I'd appreciate some reaction other than a bot telling me my issue is not interesting.

**Comment by giantkin** [comment]
Ive been watching.  I have 2 main pc's , ones working perfectly.  the other, the nextcloud client wont start up at all.  I dont know if its similiar to this or not.  (i dl'ed the latest and tried that to no avail as well.)

**Comment by FlexW** [comment]
@giantkin If it won't start, than it's likely a different issue.

@bllngr I've added the approved label, then the bot will not close it but without steps to reproduce the issue, it will be difficult to fix it.

**Comment by giantkin** [comment]
> @giantkin If it won't start, than it's likely a different issue.
> 

Unfortunately i know its something else.  Probably win10 update. (i have 2 things blocked via update now, printer is another!)

**Comment by bllngr** [comment]
I debugged the issue and it turns out that the dialog _is_ in fact shown, but for some reason the QML was not rendered. Paired with the fact that `Window.qml` disables window decorations, this resulted in a completely transparent window with no window borders.

However, since **it works as expected with `QT_OPENGL=angle`**, I suspect a graphics driver issue.

For the records, this is with a Radeon R9 380 (VEN_1002, DEV_6938) with the latest supported Radeon driver (21.5.2) on Windows 10.

At work, we switched to ANGLE by default on Windows because it's much less problematic than OpenGL. I suggest you think about that as well.
